modules/3d/test/test_pointcloud.cpp

```diff
@@ -96,11 +97,13 @@ TEST(PointCloud, LoadSavePly)
 
     std::vector<cv::Point3f> points_gold;
     std::vector<cv::Point3f> normals_gold;
+    std::vector<cv::Point3_<uchar>> rgb_gold;
 
-    cv::loadPointCloud(new_path, points_gold, normals_gold);
+    cv::loadPointCloud(new_path, points_gold, normals_gold, rgb_gold);
 
     EXPECT_EQ(normals_gold, normals);
     EXPECT_EQ(points_gold, points);
+    EXPECT_EQ(rgb_gold, rgb);
```

Comment:
The written file does not contain color information. So the test is useless in current version. Please modify exising ply or add yet another one with color information (different colors).